Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h

```diff
@@ -309,6 +309,7 @@ auto InvMassJpsiToEE(const T& candidate)
 
 // 2-prong decay candidate table
 DECLARE_SOA_TABLE(HfCandProng2Base, "AOD", "HFCANDP2BASE", //!
+                  o2::soa::Index<>,
```

Comment:
If I understand correctly, this adds an index to the 2 prong table, needed for the X building. Shouldnt we add it also to the 3 prong candidate table at this point?

Reply:
Yes, I think so. I would argue that it is best to propagate changes to the 2- and 3-prong candidate tables as consistently as possible.

Reply:
@vkucera what you suggest?